.gitignore

```diff
@@ -53,7 +53,7 @@ audit/*.json
 /.vscode/
 
 # Semantic UI
-/semantic/*
-!/semantic/src/site
-!/semantic/src/theme.config
+# /semantic/*
```

Comment:
just delete the lines i'll recommend do not only comment

Reply:
resolved via new commit